### PR TITLE
DASHCOL-3: Add Leaderboard component and added all manufacturers SVG Logos

### PR DESCRIPTION
**Pull request by marcosmonaco**
- add Leaderboard and Segment components
- add all manufacturers SVG logos
- add colors to the tailwind color palette

### CODE REVIEW

**Review by LeandroLescano — DISMISSED** [review]

File: src/components/Leaderboard/components/Segments/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+              key={`sector-${index}`}
+              className={`w-2 h-2 rounded-full ${
+                segment === 2048
+                  ? "bg-yellow-500"
+                  : segment === 2049
+                  ? "bg-green-500"
+                  : segment === 2051
+                  ? "bg-purple-500"
+                  : "bg-gray-500"
+              }`}
```
Comment (FJanza) [inline src/components/Leaderboard/components/Segments/index.tsx]: Se podria implementar classnames [link docu](https://www.npmjs.com/package/tailwindcss-classnames)
Deberia quedar algo asi:
`className={
    classnames('
        w-2 h-2 rounded-full bg-gray-500', 
        {[bg-yellow-500']: segment === 2048},
        {[bg-green-500']: segment === 2049},
        {[bg-purple-500']: segment === 2051},
    )
}`

File: src/components/Leaderboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      is_pit_out_lap: false,
+      duration_sector_1: 27.928,
+      duration_sector_2: 32.843,
+      duration_sector_3: 36.953,
+      segments_sector_1: [null, 2048, 2048, 2048, 2048, 2048],
+      segments_sector_2: [2048, 2048, 2048, 2048, 2048, 2048, 2048],
+      segments_sector_3: [2048, 2048, 2048, 2048, 2048, 2048, 2049, 2049],
+      lap_number: 25,
+    },
+  ];
```
Comment (FJanza) [inline src/components/Leaderboard/index.tsx:23]: Eliminar el dummy

File: src/components/Leaderboard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
+import SegmentDisplay from "./components/Segments";
+
+export default function Leaderboard() {
```
Comment (FJanza) [inline src/components/Leaderboard/index.tsx:3]: Se deberia pasar la data o la sessionKey y dentro hacer la lógica para obtener la data

**Review by FJanza — COMMENTED** [review]
Nice PR

**Review by FJanza — APPROVED** [review]
🚀

**Pull request merged**